tox.ini

```diff
@@ -35,7 +35,9 @@ deps =
 commands = flake8 flake8_import_order/
 
 [testenv:setuppy]
-deps = docutils
+deps =
+    docutils
+    Pygments
```

Comment:
I don't think PyPI supports `.. code-block:: python` so I'm not sure it should be used in the readme.